.eslintrc

```diff
@@ -4,5 +4,8 @@
     quotes: [2, 'single'],
     jsx-quotes: [2, 'prefer-single'],
     jsx-a11y/href-no-hash: [0],
-  }
+  },
+  globals: {
+    SASSDOC_JSON: true,
```

Comment:
Is there any way to accomplish what this is doing without needing to expose a global for it?